.github/workflows/build.yaml

```diff
@@ -96,6 +96,7 @@ jobs:
   upload:
     runs-on: ubuntu-latest
     needs: build
+    if: always()
```

Comment:
I think we need this on the last two steps of the workflow after `'Build Wolfi'` too

Reply:
I think I actually wouldn't start by changing `upload` at all.

Let's ~fork it into a new second job that only runs `if: failure()` and dry runs 🤔